packages/@remirror/core/src/builtins/persistent-selection-extension.ts

```diff
@@ -0,0 +1,50 @@
+// XXX: @remirror/core exports a _type_ 'DecorationSet', but we need access to the static methods as well.
```

Comment:
I'm not sure what this comment means. Do you want to export the `DecorationSet` from `@remirror/core`?

Reply:
`@remirror/Core` exports the `DecorationSet` and `Decoration` types, but using `DecorationSet.empty` lead to Compiler errors as that would need the static property. 

Arguably: I can remove the comment if importing from `@remirror/pm/view` is ok.

Reply:
Yeah, importing from `@remirror/pm/view` is expected. The types exported by `@remirror/core-types` are all aliases. Initially, it was to reduce the number of import statements needed when creating an extension. The types for `EditorView`, `EditorSchema`, `Selection`, `ProsemirrorNode`, `Mark` are used so often that it made sense to alias them.